sw/device/silicon_creator/lib/cert/cwt_dice_chain_entry_input.hjson

```diff
@@ -0,0 +1,48 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    name: "cwt_dice_chain_entry_input",
```

Comment:
Why are both `cwt_dice_chain_entry` and `cwt_dice_chain_entry_input` templates needed?

According to the RFC, only the former is needed: https://docs.google.com/document/d/1zBudM3YyasCoYhchRDLJUKKDEtlCQ4Yc5dx9HEE45HU/edit?usp=sharing Or if the latter (`cwt_dice_chain_entry_input`) is more configurable, then just the latter is needed, but it could be renamed `cwt_dice_chain_entry`?

Reply:
From the comment in the cddl below, the `signature` of `cwt_dice_chain_entry` is generated from `DiceChainEntryInput`. 
https://cs.android.com/android/platform/superproject/main/+/main:hardware/interfaces/security/rkp/aidl/android/hardware/security/keymint/generateCertificateRequestV2.cddl;l=130

So we use `cwt_dice_chain_entry_input` to generate the structure for hashing, and then compose them later.
https://github.com/lowRISC/opentitan/pull/24835/commits/44f1f328eb2e8cb6bde9c40c88d027f4a0cd0d0a#diff-440972298691dc2c829f4947475618f3ebba88424659d7ab75717d7f2dd77081R81

Reply:
Ah I see. Thanks. Perhaps you could add links to the CDDL code as comments in all of these HJSON files so there is a trace of where they are coming from?

Reply:
Done.